ComputeClient/Compute.Client.UnitTests/Server20/ServerAccessorTests.cs

```diff
@@ -262,5 +262,123 @@ public async Task NotifyNicIpChange_ReturnsResponse()
             Assert.AreEqual("NOTIFY_NIC_IP_ADDRESS_CHANGE", response.operation);
             Assert.AreEqual("IN_PROGRESS", response.responseCode);
         }
+
+        [TestMethod]
+        public async Task ListHistoricalServerConfigurationsPaginated_ReturnsResponse()
```

Comment:
Test used to verify that sample API response payload contained in **ListHistoricalServerConfigurationResponse.xml** is correctly marshalled to _HistoricalServerConfigurations_